SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestNetReference/SpatialTestNetReference.cpp

```diff
@@ -171,6 +178,10 @@ void ASpatialTestNetReference::BeginPlay()
 			CubeToDestroy->Destroy();
 		}
 
+		// Reset the PositionUpdateFrequency
+		GetMutableDefault<USpatialGDKSettings>()->PositionUpdateFrequency = 1.0f;
```

Comment:
Do you need to save the config ?
Also, save the previous value in a variable somewhere, to be able to restore it properly.